Formula/libvirt.rb

```diff
@@ -3,6 +3,7 @@ class Libvirt < Formula
   homepage "https://www.libvirt.org"
   url "https://libvirt.org/sources/libvirt-6.1.0.tar.xz"
   sha256 "167c185be45560e73dd3e14ed375778b555c01455192de2dafc4d0f74fabebc0"
+  revision 1
```

Comment:
I'm not sure if revision bump is necessary here, it won't uninstall unused `docutils` anyway but will force uses to redownload `libvirt` without any visible benefit.
I would remove revision bump and just update bottles (this comment for maintainer)

Reply:
Something changed in the packaging so a revision is the correct way to update the bottles iyam.

Reply:
Fair enough, I'll merge it then